WORKSPACE

```diff
@@ -110,22 +96,33 @@ load("@io_bazel_rules_sass//:defs.bzl", "sass_repositories")
 
 sass_repositories()
 
+# gRPC.
+http_archive(
+    name = "com_github_grpc_grpc",
+    sha256 = "b2f2620c762427bfeeef96a68c1924319f384e877bc0e084487601e4cc6e434c",
+    strip_prefix = "grpc-1.42.0",
+    urls = [
+        "https://github.com/grpc/grpc/archive/v1.42.0.tar.gz",  # 2021-11-17
```

Comment:
Let's also add a mirror URL here if it's not too much trouble

Reply:
Done.